### PR TITLE
adding day 2 operators code solution - tip calculations

### PR DESCRIPTION
**Pull request by RathorePoorva**

### CODE REVIEW

File: day2-ops.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+// Complete the solve function below.
+function solve(meal_cost, tip_percent, tax_percent) {
+    let tip = parseFloat(meal_cost * (tip_percent/100));
+    let tax = parseFloat(meal_cost * (tax_percent/100));
+    console.log(Math.round(meal_cost + tip + tax));
+}
+
+function main() {
+    const meal_cost = parseFloat(readLine());
```
Comment (hjaintech) [inline day2-ops.js]: Use camelCase to name variables. Make this change in all variable names in this file
For example `meal_cost` should be `mealCost`

File: day2-ops.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    main();
+});
+
+function readLine() {
+    return inputString[currentLine++];
+}
+
+// Complete the solve function below.
+function solve(meal_cost, tip_percent, tax_percent) {
+    let tip = parseFloat(meal_cost * (tip_percent/100));
```
Comment (hjaintech) [inline day2-ops.js]: No need to use `let`. These can be `const`

File: day2-ops.js (hunk 1 above)
Comment (RathorePoorva) [reply day2-ops.js]: thank you, will update it for sure :)

File: day2-ops.js (hunk 2 above)
Comment (RathorePoorva) [reply day2-ops.js]: okay, will update the changes, thanks for the help 👍

**Pull request merged**